controllers/articles.controllers.js

```diff
@@ -17,6 +18,28 @@ function getAllArticles(req,res,next){
        next(err)
     })
 
+
+
+}
+
+
+function postVoteByArticleId(req,res,next) {
+    const {article_id} = req.params
+    const {inc_votes} = req.body
+
+    getVoteCountByArticleId(article_id).then(({votes})=>{
+        const updateVotes = votes + inc_votes;
+       
+        updateVoteByArticleId(updateVotes,article_id).then((result)=> {
+            res.status(200).send({updatedArticleVoteCount:result})
+        }).catch((err)=>{
```

Comment:
remove one of these catches, you don't need two and it could potentially cause errors